doc/stdlib/linalg.rst

```diff
@@ -823,25 +823,14 @@ Linear algebra functions in Julia are largely implemented by calling functions f
 
    Construct a diagonal matrix and place ``v`` on the ``k``\ th diagonal.
 
-.. function:: scale(A, b)
-              scale(b, A)
-
-   .. Docstring generated from Julia source
-
-   Scale an array ``A`` by a scalar ``b``\ , returning a new array.
-
-   If ``A`` is a matrix and ``b`` is a vector, then ``scale(A,b)`` scales each column ``i`` of ``A`` by ``b[i]`` (similar to ``A*diagm(b)``\ ), while ``scale(b,A)`` scales each row ``i`` of ``A`` by ``b[i]`` (similar to ``diagm(b)*A``\ ), returning a new array.
-
-   Note: for large ``A``\ , ``scale`` can be much faster than ``A .* b`` or ``b .* A``\ , due to the use of BLAS.
```

Comment:
The deprecation suggestion is to use `Diagonal(b)*A` which calls BLAS. Actually, the difference is not really because of BLAS, but broadcasting overhead. With a nested loop, the code vectorizes and is as fast as BLAS on my machine.

We could move the "Note" to an entry for `*(Diagonal,Matrix)` or `broadcast`, but I'm not sure it will be easy to find in any of those places.